third_party/txt/src/txt/paragraph.h

```diff
@@ -79,7 +79,10 @@ class Paragraph {
     // The line spacing will be added to the top of the rect.
     kIncludeLineSpacingTop,
     // The line spacing will be added to the bottom of the rect.
-    kIncludeLineSpacingBottom
+    kIncludeLineSpacingBottom,
+
```

Comment:
extra space? not consistent with rest of the code.

Reply:
This is intended to separate `kStrut` from the group of `kIncludeLine` values